Formula/patchage.rb

```diff
@@ -0,0 +1,37 @@
+class Patchage < Formula
+  desc "Modular patch bay for audio and MIDI systems based on Jack and Alsa"
+  homepage "https://drobilla.net/software/ganv"
+  url "https://download.drobilla.net/patchage-1.0.0.tar.bz2"
+  sha256 "6b21d74ef1b54fa62be8d6ba65ca8b61c7b6b5230cc85e093527081239bfeda9"
+  head "https://git.drobilla.net/patchage.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "boost"
+  depends_on "ganv"
+  depends_on "glib"
+  depends_on "glibmm"
+  depends_on "gtk-mac-integration"
+  depends_on "gtkmm"
+  depends_on "jack"
+
+  needs :cxx11
+
+  def install
+    ENV.cxx11
+
+    unless build.head?
+      inreplace "src/Patchage.cpp", "GtkOSXApplication",
```

Comment:
Can we apply an upstream commit on that?

Reply:
It's fixed in the master branch upstream, not yet in release.

Reply:
We could use a `patch do` block with the upstream commit.